include/eve/conditional.hpp

```diff
@@ -293,7 +293,7 @@ namespace eve
       }
       else
       {
-        using i_t   = as_integer_t<typename type::mask_type>;
+        using i_t   = typename as_integer<typename type::mask_type>::type;
```

Comment:
Are you saying that `_t` does not work on MSVC? Can you recheck? This is bizzare.